.github/workflows/cd.yml

```diff
@@ -64,8 +64,10 @@ jobs:
 
     - name: Add final tags
       run: |
-        docker pull ghcr.io/vmware/repository-service-tuf-api:${{ github.ref_name }}
-        docker tag ghcr.io/vmware/repository-service-tuf-api:${{ github.ref_name }} latest
+        docker pull ghcr.io/vmware/repository-service-tuf-api:${{ github.sha }}
+        docker tag ghcr.io/vmware/repository-service-tuf-api:${{ github.sha }} ghcr.io/vmware/repository-service-tuf-api:latest
+        docker tag ghcr.io/vmware/repository-service-tuf-api:${{ github.sha }} ghcr.io/vmware/repository-service-tuf-api:${{ github.ref_name }}
```

Comment:
Why do you need to tag the same image as `latest` and with its `sha`?
Given that you execute the commands one after another do you there is a possibility someone will override `latest` and it won't point out to the same image?
Maybe it's good to do it that way from a security perspective.

Reply:
> Why do you need to tag the same image as `latest` and with its `sha`?
I don't tag the same image as `latest` and `sha`. I tag the `sha` with the additional tag `latest`

The `docker tag` command is `docker tag <image:tag> <image:aditional-tag>`

> Given that you execute the commands one after another do you there is a possibility someone will override `latest` and it won't point out to the same image?
> Maybe it's good to do it that way from a security perspective.

I'm reordering the commands to release first the `github.ref_name`

The latest will be added only for CD workflows. The Maintainers should handle a race condition. The process of releasing shouldn't be done without a PR approved before and alignment between the component maintainers. (we still not having the license to add the approval step in GH Actions)

Another option is to eliminate the `latest` tag for our Docker Images.

Reply:
I got what you did.
I agree we are the ones who start the release process and if we are diligent the probability of race condition is close to zero.